library/Businessprocess/BpConfig.php

```diff
@@ -150,11 +152,18 @@ public function applyChanges(ProcessChanges $changes)
         $cnt = 0;
         foreach ($changes->getChanges() as $change) {
             $cnt++;
-            $change->applyTo($this);
-        }
-        $this->changeCount = $cnt;
+            try {
+                $change->applyTo($this);
+            } catch (ConfigurationError $configError) {
+                if (!is_a($change, 'Icinga\Module\Businessprocess\Modification\NodeRemoveAction')) {
```

Comment:
Please

1. use `instanceof` instead of `is_a()`
2. don't write the full class path as you're using it above